tox.ini

```diff
@@ -90,6 +91,16 @@ commands =
     {toxinidir}/docs \
     {posargs}
 
+[testenv:schema]
```

Comment:
The schema tests are ignored when running from a temporary directory due to path mismatches. 
A tox env was added so that the `asdf_schema_root` was set to the correct path and was not run from a `.tmp` directory which ensures the schema is located and tested properly.
[Relevant chat](https://github.com/sunpy/ndcube/pull/708/files#r1624495728)